sw/device/silicon_creator/rom/rom.c

```diff
@@ -113,23 +113,40 @@ static rom_error_t rom_irq_error(void) {
   return kErrorInterrupt + mcause;
 }
 
+static inline bool rom_console_enabled(void) {
+  return otp_read32(OTP_CTRL_PARAM_OWNER_SW_CFG_ROM_CONSOLE_EN_OFFSET) !=
+         kHardenedBoolFalse;
+}
```

Comment:
This is specifically for the banner message only, right? A silicon owner might want the few milliseconds* of boot time back for the happy path, yet want to keep UART messages for faults--At first glance, it looks like that's what the code does.

(Maybe a naming nit, hehe)

*I'm not sure if I've added the pieces up correctly, but it looks like the banner is 26 characters of blocking uart_putchar() calls.

Reply:
Yes - this is to control the printing of the banner message in the ROM (and we'll presumably obey the same OTP in the ROM_EXT).  Disabling it allows the owner to reclaim the time spent printing the banner message (currently 46 characters including the newlines (Looks like about 4ms at 115200bps).

I renamed the OTP from `ROM_CONSOLE_EN` to `ROM_BANNER_EN`, as this OTP does not affect printing of boot fault information.

Reply:
Awesome, thanks!